### DISCUSSION TITLE
supabase.auth.admin undefined

### DISCUSSION THREAD

**Discussion by brucemclachlan** [supabase/supabase#14563] [Questions]
I just started getting 'undefined' errors for supabase.auth.admin

Is anyone else seeing this?

Using supabase-js

UPDATE: upgrading from v1 to v2 seems to have fixed this

**Comment by GaryAustin1** [comment]
That would not have worked at all on v1 supabase-js.  The "just started getting", does not make much sense in that context as it would only start failing if you changed to admin versus api.   Edit: or I guess went back to v1 from v2.
